spec/unit/app/models/edition/workflow_completion_spec.rb

```diff
@@ -0,0 +1,59 @@
+RSpec.describe Edition::WorkflowCompletion do
+  let(:organisation) { build(:organisation) }
+  let(:schema) { build(:schema, block_type: "content_block_type", body: { "properties" => { "foo" => "", "bar" => "" } }) }
+  let(:document) { create(:document, :pension, id: 567) }
+  let(:edition) do
+    build(:edition,
+          id: 123,
+          document: document,
+          lead_organisation_id: organisation.id)
+  end
+
+  before do
+    allow(Schema).to receive(:find_by_block_type).and_return(schema)
```

Comment:
I guess if you get the `schema` directly from the `edition` then you might stub `edition#schema` instead?

Reply:
This is a good idea. However, if I remove this the PublishEditionService then errors. I'm happy to remove the reliance on the find_by_block_type class method but I think I should do it in a future PR.